File: Soe.Common/DTO/EmployeeEarnedHolidayDTO.cs

using SoftOne.Soe.Common.Attributes;

namespace SoftOne.Soe.Common.DTO
{
    [TSInclude]
    public class EmployeeEarnedHolidayDTO
    {
        public int EmployeeId { get; set; }
        public string EmployeeNr { get; set; }
        public string EmployeeName { get; set; }
        public decimal EmployeePercent { get; set; }
        public bool Work5DaysPerWeek { get; set; }
        public string Work5DaysPerWeekString { get; set; }
        public bool HasTransaction { get; set; }
        public string HasTransactionString { get; set; }
        public bool? Suggestion { get; set; }
        public string SuggestionString { get; set; }
        public string SuggestionNote { get; set; }
    }
}

File: Soe.Angular/TypeScript/Common/Models/EmployeeEarnedHolidayDTO.ts
import { IEmployeeEarnedHolidayDTO } from "../../Scripts/TypeLite.Net4";

export class EmployeeEarnedHolidayDTO implements IEmployeeEarnedHolidayDTO {
    employeeId: number;
    employeeName: string;
    employeeNr: string;
    employeePercent: number;
    hasTransaction: boolean;
    hasTransactionString: string;
    suggestion: boolean;
    suggestionNote: string;
    suggestionString: string;
    work5DaysPerWeek: boolean;
    work5DaysPerWeekString: string;
}
